### PR TITLE
Added Norwegian translation for the Eclipse Collections website

### PR DESCRIPTION
**Pull request by mehmandarov**
Changelog:

* Added Norwegian translation for the Eclipse Collections website
* Updated readme file with the link to the Norwegian translation. 

TODO:
* ~~I will tag a reviewer for the translation shortly.~~ – _Done ✅_
* ~~I will update draft release notes once this PR is accepted.~~

### CODE REVIEW

**Comment by nikhilnanivadekar** [comment]
Thanks for the contribution! Don’t worry about updating the release notes drafts. I’ll take care of it when I compose all the notes.

**Comment by mehmandarov** [comment]
@nikhilnanivadekar: My pleasure! Ok, I will not add any changes to release notes yet.

**Comment by mehmandarov** [comment]
Did a second pass and added some minor changes and language cleanup. It should be ready for peer review. Can you have a look at it please, @madsop?

File: site/no-nb/index.html [hunk 1]
```diff
@@ -0,0 +1,669 @@
...
+                        noensinne – det passer perfekt når du skal utvikle din Java-kode.
+                        <br><br>Integrer den i kodebasen din i dag med dine favorittbyggverktøy!</p>
+                    <object id="badge" data="https://snyk-widget.herokuapp.com/badge/mvn/org.eclipse.collections/eclipse-collections/badge.svg" type="image/svg+xml"></object>
+                </div>
+                <div class="col-lg-7 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#maven" data-toggle="tab">Maven</a></li>
+                        <li><a href="#gradle" data-toggle="tab">Gradle</a></li>
+                        <li><a href="#ivy" data-toggle="tab">Ivy</a></li>
+                        <li><a href="#download" data-toggle="tab">Nedlasting</a></li>
```
Comment (madsop) [inline site/no-nb/index.html:165]: I would rather say "Last ned", to use a more active language.

File: site/no-nb/index.html [hunk 2]
```diff
@@ -0,0 +1,669 @@
...
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Kom i gang med <br>Eclipse Collections</h2>
+                    <p class="lead">Eclipse Collections er det beste rammeverket for Java-samlinger 
+                        noensinne – det passer perfekt når du skal utvikle din Java-kode.
```
Comment (madsop) [inline site/no-nb/index.html:156]: Sounds more natural to me with "Java-koden din" than "din Java-kode"

File: site/no-nb/index.html [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-7 col-lg-5 col-sm-push-6  col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Rikelige, konsise og lesbare API-er</h2>
+                    <p class="lead">Maksimer kraften til lambda-uttrykk og metodereferanser i Java 8 med rikelige API-er
```
Comment (madsop) [inline site/no-nb/index.html:218]: Somewhat unsure on this one - is "omfattende" more suitable than "rikelige"?

File: site/no-nb/index.html [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+                GS Collections har blitt presentert på en rekke
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">konferanser</a>,
+                inkludert JVM Summit i 2012 og JavaOne i 2014.
+                En sammenligning av ytelse mellom parallelle lazy implementasjoner i Java 8, Scala og GS Collections ble presentert på
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> i 2014.
+                Videre ble <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">artikler</a> om GS Collections
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">Del 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">Del 2</a>)
+                publisert på InfoQ.com.
+                Disse artikler viser noen av funksjonene i rammeverket gjennom eksempler, samt et intervju med skaperen av GS Collections.
```
Comment (madsop) [inline site/no-nb/index.html:522]: I'd say "Disse artiklene"

File: site/no-nb/index.html [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+    <a class="anchor" name="contribute"></a>
+    <div class="content-footer">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-4 col-sm-4">
+                    <h3>Bidra</h3>
+                    <p class="regular">
+                        Vi tar imot bidrag med glede!
```
Comment (madsop) [inline site/no-nb/index.html:571]: I'd say either "Vi tar med glede imot bidrag" or "Vi tar gjerne imot bidrag", perhaps preferably the first one, sounds a bit more natural.

File: site/no-nb/index.html [hunk 6]
```diff
@@ -0,0 +1,669 @@
...
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">Del 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">Del 2</a>)
+                publisert på InfoQ.com.
+                Disse artikler viser noen av funksjonene i rammeverket gjennom eksempler, samt et intervju med skaperen av GS Collections.
+                <br><br>
+
+                Gjennom årene har rundt 40 utviklere fra det samme selskapet bidratt til rammeverket.
+                <br><br>
+
+                For å maksimere prosjektets potensiale som et åpen kildekode-prosjekt, ble GS Collections migrert til
```
Comment (madsop) [inline site/no-nb/index.html:528]: _Maksimere_ or _utnytte_? Or even _Fullt ut utnytte_?

File: site/no-nb/index.html [hunk 7]
```diff
@@ -0,0 +1,669 @@
...
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Mange containertyper, inkludert <br>uforanderlige samlinger, primitive samlinger, Bimap, Multimap og Bag</h2>
+                    <p class="lead">Uforanderlige (immutable) samlinger, samlinger av primitive typer, Bimap (omvendt ordbok hvor du kan finne en nøkkel etter verdi), 
+                        Multimap (multi-ordbøker – ordbøker med ikke-unike nøkler) og Bag (flersett – et sett med gjentagende elementer) – du kan bruke alle 
+                        disse typer containere ved bruk av praktiske og lett tilgjengelige factory-metoder.</p>
```
Comment (madsop) [inline site/no-nb/index.html:300]: "disse typene" sounds better to me.

**Review by madsop — CHANGES_REQUESTED** [review]

**Comment by mehmandarov** [comment]
Thanks @madsop! 🙌  I will have a look at it latest tomorrow!

**Comment by nikhilnanivadekar** [comment]
@mehmandarov if you pull the recent changes, you will see that @bmvermeer added badges, so please make sure you incorporate those in the translation as well.

File: site/no-nb/index.html [hunk 8]
```diff
@@ -0,0 +1,669 @@
+<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2020 Goldman Sachs and others
```
Comment (nikhilnanivadekar) [inline site/no-nb/index.html:3]: Please update the copyright according to your ECA.

File: site/no-nb/index.html [hunk 9]
```diff
@@ -0,0 +1,669 @@
...
+                            <li><a href="../javadoc/8.2.0">8.2.0</a></li>
+                            <li><a href="../javadoc/8.1.0">8.1.0</a></li>
+                            <li><a href="../javadoc/8.0.0">8.0.0</a></li>
+                            <li><a href="../javadoc/7.1.0">7.1.0</a></li>
+                        </ul>
+                    </li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                           aria-expanded="false">Språk<span class="caret"></span></a>
+                        <ul class="dropdown-menu">
```
Comment (nikhilnanivadekar) [inline site/no-nb/index.html:102]: The paths need to be relative.
```
<ul class="dropdown-menu">
                        <li><a href="../index.html">English</a></li>
                        <li><a href="../de/index.html">Deutsch</a></li>
                        <li><a href="../el/index.html">Ελληνικά</a></li>
                        <li><a href="../es/index.html">Español</a></li>
                        <li><a href="../cn/index.html">中文</a></li>
                        <li><a href="">Français</a></li>
                        <li><a href="../ja/index.html">日本語</a></li>
                        <li><a href="../pt-br/index.html">Português-Brasil</a></li>
                        <li><a href="../ru/index.html">Русский</a></li>
                        <li><a href="../hi/index.html">हिंदी</a></li>
                    </ul>
```

**Review by nikhilnanivadekar — CHANGES_REQUESTED** [review]
Thanks for your contribution @mehmandarov . I added a couple of review comments, if you could fix them when you are doing the changes suggested during the review.

**Comment by mehmandarov** [comment]
Will do, @nikhilnanivadekar!

File: site/no-nb/index.html (hunk 9 above)
Comment (mehmandarov) [reply site/no-nb/index.html:102]: Good catch! Thanks! Using the English version as a starting point was not so good idea. Had to fix paths several other places too. 😃

File: site/no-nb/index.html (hunk 1 above)
Comment (mehmandarov) [reply site/no-nb/index.html:165]: Agreed! Active voice is better! Tnx

File: site/no-nb/index.html (hunk 2 above)
Comment (mehmandarov) [reply site/no-nb/index.html:156]: Yes, it has better flow to it!

File: site/no-nb/index.html (hunk 3 above)
Comment (mehmandarov) [reply site/no-nb/index.html:218]: Not really sure about this one... I kind of feel "omfattende" has a lightly more of a negative cling to it? WDYT, @madsop?

File: site/no-nb/index.html (hunk 4 above)
Comment (mehmandarov) [reply site/no-nb/index.html:522]: Agree!

File: site/no-nb/index.html (hunk 8 above)
Comment (mehmandarov) [reply site/no-nb/index.html:3]: Done, thanks!

File: site/no-nb/index.html (hunk 5 above)
Comment (mehmandarov) [reply site/no-nb/index.html:571]: Agreed!

File: site/no-nb/index.html (hunk 6 above)
Comment (mehmandarov) [reply site/no-nb/index.html:528]: Yes! _"For å utnytte fullt ut prosjektets potensiale..."_.

File: site/no-nb/index.html (hunk 7 above)
Comment (mehmandarov) [reply site/no-nb/index.html:300]: Agreed!

File: site/no-nb/index.html (hunk 3 above)
Comment (mehmandarov) [reply site/no-nb/index.html:218]: Maybe _"omfangsrike"_?

**Comment by mehmandarov** [comment]
Squashed commits as pr. request from @nikhilnanivadekar. 👍

**Comment by donraab** [comment]
Hi @mehmandarov @nikhilnanivadekar, it seems like all that is needed here is to rebase and then this can be merged. Sound good?

**Comment by mehmandarov** [comment]
Sounds good to me, @donraab!

**Comment by nikhilnanivadekar** [comment]
@mehmandarov can you please rebase?

**Comment by mehmandarov** [comment]
Done, @nikhilnanivadekar! 🎉

**Review by donraab — APPROVED** [review]
LGTM.

**Review by nikhilnanivadekar — APPROVED** [review]
LGTM

**Pull request merged**